lunatrace/bsl/frontend/src/pages/vulnerabilities/Main.tsx

```diff
@@ -52,16 +51,16 @@ export const VulnerabilitiesMain: React.FunctionComponent = () => {
   // RUN SEARCH QUERY
   const { data, isFetching, refetch } = api.useSearchVulnerabilitiesQuery({
     search: postgresSearch,
-    namespace: postgresFilter,
+    source: postgresFilter,
     order_by: postgresOrderMap[orderBy],
     limit: vulnLimit,
   });
 
   // lazy loading. Reloads all the old vulns when expanding the batch size but..it works fine
   useBottomScrollListener(
     () => {
-      if (data && data.vulnerabilities) {
-        const vulnCount = data.vulnerabilities.length;
+      if (data && data.vulnerability) {
```

Comment:
there's more than one vulnerability..im confused why this property name changed and why hasura did that